crates/roc_std/src/roc_box.rs

```diff
@@ -69,16 +71,15 @@ where
     }
 
     fn storage(&self) -> &Cell<Storage> {
-        let alignment = Self::alloc_alignment();
+        unsafe { &*self.as_ptr().cast::<Cell<Storage>>() }
+    }
 
-        unsafe {
-            &*self
-                .contents
-                .as_ptr()
-                .cast::<u8>()
-                .sub(alignment)
-                .cast::<Cell<Storage>>()
-        }
+    /// The raw pointer to a roc box, including the leading refcount
+    /// Intended for use by platforms in roc_dealloc
+    pub unsafe fn as_ptr(&self) -> *mut c_void {
```

Comment:
```suggestion
    pub unsafe fn as_refcount_ptr(&self) -> *mut c_void {
```

Want to make this clear that it is the refcount pointer and not the pointer to the data. Another reasonable name would be `as_allocation_ptr`.

Reply:
I pushed the button, then took off your co-author line when rebasing because it looked like github was unhappy about the signature.